erratatool.yml

```diff
@@ -158,5 +182,5 @@ boilerplates:
       For more details about the security issue(s), including the impact, a CVSS
       score, acknowledgments, and other related information, refer to the CVE page(s)
       listed in the References section.
-    solution: *common_solution
+    solution: *common_image_solution
```

Comment:
Could you confirm the solution template change here for CVE advisory is expected?

Reply:
For reference: https://github.com/openshift/ocp-build-data/blob/714b1477fe37c97e87714fce138ca62cb2a5849e/erratatool.yml#L186